Exercise 9.js

```diff
@@ -1,7 +1,9 @@
-const calculateMatrixSum = matrix => {
-    let sum = 0;
-    for (let i = 0; i < matrix.length; i++){
-        for (let j = 0; j < matrix[i].length; j++){
-         sum += matrix[i][j]; }}    
-    return sum; 
-}
+const calculateMatrixSum = (matrix) => {
```

Comment:
When arow function has only one parameter the parentheses can be omitted. It improves code readability.